SocialFilter/sections/specific.txt

```diff
@@ -403,7 +403,7 @@ www-news18-com.cdn.ampproject.org##.story-social-share-fix > ul
 ndtv.com,www-ndtv-com.cdn.ampproject.org##.story_ft-nav
 republicworld.com##.share-social-media
 republicworld.com,www-republicworld-com.cdn.ampproject.org##.s_p_wrapper > div[style="display:flex"]
-joemonster.org##.mobile-share-button
+mel.fm,joemonster.org##.mobile-share-button
```

Comment:
It's better to use `mobile-share-menu` class here.

for example here share popup is opened initially and your rule doesn't hide it.
https://mel.fm/podrostki/4825319-depression